PixiParticles/paket.references

```diff
@@ -0,0 +1,3 @@
+FSharp.Core
+Fable.Core
+Fable.Import.Browser
```

Comment:
I think PixiParticles don't need a deps over Fable.import.Browser as it doesn't use any type coming from this namespace.

I just read the code didn't check with VSCode so can be wrong

Reply:
ok should be good now :)